samples/java-customer-registry/src/main/java/customer/Main.java

```diff
@@ -52,7 +52,7 @@ public static void main(String[] args) throws Exception {
           // end::register[]
           .lowLevel()
           .registerValueEntity(
-              context -> new CustomerValueEntityHandler(new CustomerValueEntity()),
+              context -> new CustomerValueEntityHandler(),
```

Comment:
Wouldn't it be convenient to give the user an opportunity to pass additional things to their `CustomerValueEntity` here?

Reply:
constructor with entity is back